kevinb5617/ResuMake
# Add Navigation Buttons

# Description

According to the mockups in the `docs` directory, there should be navigation buttons to move to/from the different pages in the setup process.

# Related Issue(s) and Pull Request(s)

closes #15

# Solution

The Button component was created with a few properties to specify the destination path, type (for styling), and text.

# Steps to Test

1. `git fetch upstream`
2. `git checkout feature/create-nav-buttons`
3. `cd app`
4. `npm install`
5. `npm start`
6. After clicking the "Get Started" button, you should see the two navigation buttons on the bottom, with the left button going back to the homepage and the right button changing the web path, but not navigating to a new page, which is not implemented yet.

# Known Issues

NONE

# Checklist
